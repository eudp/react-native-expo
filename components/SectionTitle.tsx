import { StyleSheet, TextStyle } from "react-native";
import CustomText from "./CustomText";

interface Props {
  title: string;
  style?: TextStyle;
}

const SectionTitle = ({ title, style = {} }: Props) => (
  <CustomText style={[styles.sectionTitle, style]}>{title}</CustomText>
);

const styles = StyleSheet.create({
  sectionTitle: {
    color: "#9B9898",
    fontFamily: "Avenir-Black",
    fontSize: 14,
    lineHeight: 19,
    marginBottom: 15,
  },
});

export default SectionTitle;
