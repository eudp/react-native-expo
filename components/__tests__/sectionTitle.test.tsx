import React from "react";
import renderer from "react-test-renderer";
import SectionTitle from "../SectionTitle";

describe("SectionTitle", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SectionTitle title="Test Title" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
