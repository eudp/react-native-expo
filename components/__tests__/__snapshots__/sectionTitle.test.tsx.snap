// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle renders correctly 1`] = `
<Text
  style={
    [
      {
        "fontFamily": "Avenir-Regular",
      },
      [
        {
          "color": "#9B9898",
          "fontFamily": "Avenir-Black",
          "fontSize": 14,
          "lineHeight": 19,
          "marginBottom": 15,
        },
        {},
      ],
    ]
  }
>
  Test Title
</Text>
`;
